app/javascript/src/components/commons/ToolBar/index.jsx

```diff
@@ -8,12 +8,15 @@ const Toolbar = ({
   title,
   onSearchChange,
   buttonLabel,
+  handleActionButtonClick,
 }) => (
   <Header
-    actionBlock={<Button label={buttonLabel} />}
     className="px-4 py-2"
-    menuBarToggle={function noRefCheck() {}}
+    menuBarToggle={() => {}}
```

Comment:
You can create a function called `noop` in the `utils.jsx` file, of the desired scope, to define this function.  `noop` stands for `no operation` and is the general name used for functions that do not do anything. This way you can properly reuse them wherever required.